support-frontend/conf/routes

```diff
@@ -97,6 +97,7 @@ GET  /subscribe/digital/checkout/thankyou-existing  controllers.DigitalSubscript
 
 GET  /weekly                            controllers.Subscriptions.weeklyGeoRedirect()
 GET  /subscribe/weekly                            controllers.Subscriptions.weeklyGeoRedirect()
+GET  /subscribe/weekly/checkout                            controllers.WeeklySubscription.displayForm()
```

Comment:
add a query param to this route, eg. `enableCheckout` and if it's not present in the url redirect to the old version of the checkout on subscribe.

Reply:
couldn't work out how to do a query param quickly and I'm going now, so I've done two separate routes for now. but we can change it in this/another PR later.

Reply:
See an example of a similar query param in #1701 

specifically: 
https://github.com/guardian/support-frontend/pull/1701/files#diff-ccba6c5f6c4141484e7cdca2add29512L69
and 
https://github.com/guardian/support-frontend/pull/1701/files#diff-afa248f183c4090b6973e10445c72c4aL111

Reply:
thanks. I notice that one makes you log in before doing the redirect which is what I why I ended up with two routes.

Reply:
See comment below